openapi/p4pa-pu-bff.openapi.yaml

```
@@ -0,0 +1,106 @@
+openapi: 3.0.1
+info:
+  title: API Documentation
+  description: OpenAPI documentation
+  version: 1.0.0
+servers:
+  - url: "http://localhost:8080"
```

let's introduce a specialized base path
```suggestion
  - url: "http://localhost:8080/bff"
```